pyproject.toml

```diff
@@ -3,6 +3,164 @@
 requires = ['setuptools', 'wheel', 'plux>=1.7']
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "localstack-core"
+authors = [
+        { name = "LocalStack Contributors", email = "info@localstack.cloud" }
+]
+description = "The core library and runtime of LocalStack"
+requires-python = ">=3.8"
```

Comment:
This change is the one which makes the second commit (adding all these trailing commas) necessary:
- `black` did not read this from the `setup.cfg`: https://github.com/psf/black/issues/3859#issuecomment-1699370363
- `black` adds a trailing comma for `kwargs` for 3.6+: https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html